src/extlibs/hrvo/path_point.cpp

```diff
@@ -0,0 +1,17 @@
+#include "extlibs/hrvo/path_point.h"
+
+#include <stdexcept>
+#include <utility>
+
+#include "extlibs/hrvo/vector2.h"
+
+
+PathPoint::PathPoint(const Vector2 &position)
```

Comment:
It seems like this would change once https://github.com/UBC-Thunderbots/Software/pull/2493?  It seems like the Path protobuf is lacking speed information, so where would it get that information? Would it be something AI needs to tell HRVO? It also seems like somehow the same datatype should be used between AI and HRVO

Reply:
If #2493 gets merged in first, I will need to change these references. Path point is different from the path protobuf, it's only used in the local HRVO simulator. Currently this pathpoint sets the speed to `0.0f`